src/main/java/io/devlabs/keytree/domains/company/domain/UserRole.java

```diff
@@ -0,0 +1,18 @@
+package io.devlabs.keytree.domains.company.domain;
+
+public enum UserRole {
+    MASTER("MASTER"),
+    ADMIN("ROLE_ADMIN"),
+    USER("ROLE_USER")
+    ;
```

Comment:
```suggestion
    MASTER("MASTER"),
    ADMIN("ADMIN"),
    USER("USER");
```
- 7번 라인 세미콜론 오탈자 정리 해주시고 ADMIN, USER로 가죠 MASTER는 Prefix 안붙으니까 이상한데